package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "leaderboard-app-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "start": "webpack serve --open"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/kimothokamau/leaderboard-app-api.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/kimothokamau/leaderboard-app-api/issues"
+  },
+  "homepage": "https://github.com/kimothokamau/leaderboard-app-api#readme",
+  "devDependencies": {
+    "@babel/core": "^7.14.8",
+    "@babel/eslint-parser": "^7.14.7",
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^5.2.7",
+    "eslint": "^7.31.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.23.4",
+    "hint": "^6.1.4",
+    "html-webpack-plugin": "^5.3.2",
+    "style-loader": "^3.1.0",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.20.1",
+    "webpack": "^5.44.0",
+    "webpack-cli": "^4.7.2",
+    "webpack-dev-server": "^3.11.2"
```

Comment:
- [ ] Please include the `webpack.config.js` file along with the bundled `dist` folder and also an `index.js` file in the `src` directory.

Reply:
Thanks for this feedback. I have included the suggested files.